src/main/java/nopol_examples/nopol_example_3/NopolExample.java

```diff
@@ -6,13 +6,13 @@
 	 * Return true if a is odd number
 	 */
 	public boolean isOddNumber(int a){
-		int tmp = (a-1)%2;
-		int foo = 0;
+		int tmp = (a-1)%2 ;
```

Comment:
This line (9) has been identified with a suspiciousness value of 53.88%.

<details>
     <summary>Failing tests that cover this line</summary>

- `nopol_examples.nopol_example_3.NopolExampleTest#test3`
- `nopol_examples.nopol_example_3.NopolExampleTest#test4`
- `nopol_examples.nopol_example_3.NopolExampleTest#test1`
- `nopol_examples.nopol_example_3.NopolExampleTest#test2`
- `nopol_examples.nopol_example_3.NopolExampleTest#test7`
- `nopol_examples.nopol_example_3.NopolExampleTest#test8`
- `nopol_examples.nopol_example_3.NopolExampleTest#test5`
- `nopol_examples.nopol_example_3.NopolExampleTest#test6`
- `nopol_examples.nopol_example_3.NopolExampleTest#test9`
</details>